serviceRegistry.go

```diff
@@ -95,27 +98,30 @@ func NewCocoServiceRegistry(etcd EtcdHealthCheckKeysAPI, vulcandAddr string, che
 	services := make(map[string]Service)
 	categories := make(map[string]Category)
 	measuredServices := make(map[string]MeasuredService)
-	return &EtcdServiceRegistry{etcd, vulcandAddr, checker, services, categories, measuredServices}
+	return &EtcdServiceRegistry{sync.Mutex{}, etcd, time.Duration(60) * time.Second, vulcandAddr, checker, services, categories, measuredServices}
```

Comment:
I was about to question passing the mutex by value, but then I saw it's embedded... nice